templates/pages/management/license_progressbar.html.twig

```diff
@@ -31,7 +31,7 @@
  #}
 
 <div class="position-relative w-100 d-flex">
-
+    {% set total = total|number_format %}
```

Comment:
`number_format` will return a string, but it is used on the next line for an arythmetic operation and again on the next line for a comparison.
What is the purpose of the formatting here ?